tailwind.config.js

```diff
@@ -13,7 +15,7 @@ module.exports = {
   theme: {
     container: {
       center: true,
-      padding: '1rem',
+      padding: '1.25rem',
```

Comment:
Making scrollbar appear on top of body (using overflow: overlay) needs to push content a little bit otherwise it will be hidden under scrollbar.
Figured it's best to make sure container is 4px bigger than trying to find every case by hand now (and then maybe in the future) and adding padding there